dmapool is currently used only by pci, so do not compile it for
non-pci systems. Is contained in 2.6.3-rc1-mm1, so can be removed once
it is merged from there.

			hare 2004/02/12

--- linux-2.6.2.orig/drivers/base/Makefile	2004-02-04 04:44:21.000000000 +0100
+++ linux-2.6.2/drivers/base/Makefile	2004-02-12 10:39:04.195357921 +0100
@@ -3,6 +3,7 @@
 obj-y			:= core.o sys.o interface.o bus.o \
 			   driver.o class.o class_simple.o platform.o \
-			   cpu.o firmware.o init.o map.o dmapool.o
+			   cpu.o firmware.o init.o map.o
+obj-$(CONFIG_PCI)	+= dmapool.o
 obj-y			+= power/
 obj-$(CONFIG_FW_LOADER)	+= firmware_class.o
 obj-$(CONFIG_NUMA)	+= node.o
